Documentation/guideline/csharp_coding_standards.md

```diff
@@ -112,24 +118,24 @@ The class definition contains class members in the following order, from *less*
 ### Cross-platform coding
 Our code should supports multiple operating systems. Don't assume we only run (and develop) on Windows. Code should be sensitvie to the differences between OS's. Here are some specifics to consider.
 
-* **DO** use `Enviroment.NewLine` instead of hard-coding the line break instead of `\r\n`, as Windows uses `\r\n` and OSX/Linux uses `\n`.
+* **DO** use `Enviroment.NewLine` instead of hard-coding the line break, as Windows uses `\r\n` and OSX/Linux uses `\n`.
 
-> **Note**
->
-> Be aware that thes line-endings may cause problems in code when using `@""` text blocks with line breaks.
+  > **Note**
+  >
+  > Be aware that thes line-endings may cause problems in code when using `@""` text blocks with line breaks (it spans over multiple lines).
```

Comment:
I understood the **`@""` text blocks with line breaks** as a string literal that contains hard-coded   newline character, eg. `@"text\ntext"`.

I wanted to also point out a case when `@""` is used in a multiline string literal, like this:

```csharp
var x = @"line1
line2";

Assert.AreEqual("line1\nline2", x);
```

That's why I used the "spans over multiple lines" expression to explicitly denote this case.

Is this a wrong way to call it like that?